modular_splurt/code/game/objects/effects/decals/cleanable/lewd_decals.dm

```diff
@@ -0,0 +1,31 @@
+/datum/reagent/consumable/milk/reaction_turf(turf/T, reac_volume)
+	if(!istype(T))
+		return
+	if(reac_volume < 3)
+		return
```

Comment:
Isn't there already code to handle milk splashes in the code? I'm pretty sure I've seen splashed milk before